src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,38 @@
+@mixin regularTex($lineHeight: 14px, $fontSize: 10px) {
+  font-size: $fontSize;
+  line-height: $lineHeight;
+  font-weight: 400;
+}
+
+@mixin normalTex($lineHeight: 18px, $fontSize: 12px) {
+  font-size: $fontSize;
+  line-height: $lineHeight;
+  font-weight: 500;
+}
+
+@mixin boldTex($lineHeight: 18px, $fontSize: 16px) {
+  font-size: $fontSize;
+  line-height: $lineHeight;
+  font-weight: 700;
+}
+
+@mixin flexProperties($alignItems: center, $justifyContent: space-between, $flexDirection: column) {
+  display: flex;
+  align-items: $alignItems;
+  justify-content: $justifyContent;
+  flex-direction: $flexDirection;
+}
+
+@mixin starRegular {
+  background-image: $starRegular;
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
+  // background-position: center;
+}
+
+@mixin starActive {
+  background-image: $starActive;
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
```

Comment:
Try to refactor the code to not repeat the same code for active and regular stars. I think you don't need a mixin for it, you can just change the bg image